net/unbound/files/unbound.init

```diff
@@ -6,6 +6,9 @@ START=61
 USE_PROCD=1
 
 start_service() {
+	chmod -R u=rwX,g=rX,o= /etc/unbound
+	chown -R unbound:unbound /etc/unbound
+
```

Comment:
Okay, I see.  But I am afraid that the chmod/chown call will cause writing to flash storage which I really want to prevent as much as possible.

Can you rework it in a way resembling what `net/seafile-server` does in the `postinst` script?